spec/lib/quickbooks/service/base_service_spec.rb

```diff
@@ -187,12 +191,24 @@
       Quickbooks.log = true
       Quickbooks.log_xml_pretty_print = false
       expect_any_instance_of(Nokogiri::XML::Document).not_to receive(:to_xml)
-      expect(Quickbooks.logger).to receive(:info).at_least(1)
+      expect(Quickbooks.logger).to receive(:info).exactly(10)
       @service.query
       Quickbooks.log = false
       Quickbooks.log_xml_pretty_print = true
     end
 
+    it 'should log once for request and once for response if Quickbooks.condense_logs = true' do
```

Comment:
Can use `context`.

Reply:
agreed, but I feel like introducing context blocks here adds a new pattern to the testing sweet that's should be applied more generally rather than as a 1 off, which is why I stuck w the it block